tests/test_edge_cases.py

```diff
@@ -89,3 +90,32 @@ def test_non_negative_counts():
 
     with pytest.raises(ValueError):
         DeseqDataSet(counts_df, clinical_df, design_factor="condition")
+
+
```

Comment:
Could you also test that a key error is raised when the ref level isn't in the design factor values ? I think it will be the most common error

Reply:
Done in [0bdcd67](https://github.com/owkin/PyDESeq2/pull/26/commits/0bdcd67dcb46bd35ff7e27fcb3862c09abe2fbc1)